doc/ref/chap-type-method.md

```diff
@@ -63,7 +63,9 @@ Respects unicode.
 
 ### extend()
 
-### find()
+### indexOf()
+
+Returns -1 if element is not in the index.
```

Comment:
How about

"Returns the first index of the element in the List, or -1 if it's not present."

 "element is not in the index" is confusing

Sorry I didn't catch this the first time around

Reply:
should've been "element is not in the List" but your proposal is probably a bit more clear :)